docs/devGuide/devGuide.md

```diff
@@ -136,20 +136,37 @@ $ npm run test
 For Windows users:
 
 ```
-$ npm run testwin
+> npm run testwin
 ```
 
 #### Updating tests
 
-When adding new features, updating existing features or fixing bugs, you should update the expected site to reflect the changes.
+Whether you are adding a new feature, updating existing features or fixing bugs, make sure to update the test sites to reflect the changes.
 
-##### Changes to existing features
+You may use the following script to do this:
 
-Simply update the expected HTML files in `test/test_site/expected/` to reflect the changes.
+For Unix:
+
+```
+$ npm run updatetest
+```
+
+For Windows users:
```

Comment:
Maybe standardize to "On Unix" and "On Windows" like you mentioned in the maintainer guide?

Reply:
Done!